Calculator/Calculator.cpp

```diff
@@ -328,6 +126,25 @@ void add_sub_calc(vector<string> &equ_vec) {
                                         //Otherwise, there are no more operators in the equation, found_operator remains false, loop ends
 }
 
+string remove_spaces(string& old_str) {

```

Comment:
guess i'll be consistently pedantic and point it out here too: `string &old_str`